Tests/Unit/Util/LLRBTreeTests.swift

```diff
@@ -30,41 +30,69 @@ class RedBlackTreeTests: XCTestCase {
 
     func test_can_put_and_remove_while_keeping_order() {
         for array in self.sources {
-            let target = RedBlackTree<Int, Int>()
+            let target = LLRBTree<Int, Int>()
             for value in array {
-                target.insert(key: value, value: value)
+                target.insert(value, value)
             }
 
             XCTAssertEqual(target.minValue(), 0)
             XCTAssertEqual(target.maxValue(), 9)
 
             XCTAssertEqual(target.allValues(), [0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
 
-            target.delete(key: 8)
+            target.delete(8)
             XCTAssertEqual(target.allValues(), [0, 1, 2, 3, 4, 5, 6, 7, 9])
 
-            target.delete(key: 2)
+            target.delete(2)
             XCTAssertEqual(target.allValues(), [0, 1, 3, 4, 5, 6, 7, 9])
 
-            target.delete(key: 5)
+            target.delete(5)
             XCTAssertEqual(target.allValues(), [0, 1, 3, 4, 6, 7, 9])
         }
     }
 
     func test_can_query_floor_entry() {
         for array in self.sources {
-            let target = RedBlackTree<Int, Int>()
+            let target = LLRBTree<Int, Int>()
             for value in array {
-                target.insert(key: value, value: value)
+                target.insert(value, value)
             }
 
-            XCTAssertEqual(target.floorEntry(input: 8), 8)
+            XCTAssert(target.floorEntry(-1) == nil)
 
-            target.delete(key: 8)
-            XCTAssertEqual(target.floorEntry(input: 8), 7)
+            for index in 0 ..< 9 {
+                XCTAssertEqual(target.floorEntry(index)!.1, index)
+            }
+
+            XCTAssertEqual(target.floorEntry(8)!.1, 8)
+
+            target.delete(8)
+            XCTAssertEqual(target.floorEntry(8)!.1, 7)
+
+            target.delete(7)
+            XCTAssertEqual(target.floorEntry(8)!.1, 6)
+
+            target.delete(6)
+            target.delete(4)
+            XCTAssertEqual(target.floorEntry(8)!.1, 5)
+
+            XCTAssertEqual(target.floorEntry(5)!.1, 5)
+
+            XCTAssertEqual(target.floorEntry(4)!.1, 3)
+
+            target.insert(4, 4)
+            XCTAssertEqual(target.floorEntry(4)!.1, 4)
+        }
+    }
+
+    func test_insert() {
+        for array in self.sources {
+            let target = LLRBTree<Int, Int>()
+            for value in array {
+                target.insert(value, value)
+            }
 
-            target.delete(key: 7)
-            XCTAssertEqual(target.floorEntry(input: 8), 6)
+            target.printNode()
```

Comment:
Since we can't check logs every time in CI, wouldn't it be better to leave this assertion?

Reply:
The purpose of test_insert() is to build a tree and visualize tree structures for debugging. So delete and floorEntiry are not relevant.
And floorEntity tests are performed in  test_can_query_floor_entry.